### ISSUE TITLE
Changes to tiddlers are not saved

### ISSUE THREAD

**Issue by adamschmideg** [bmann/tiddlywiki-heroku#2]
I try to run it locally with `npm start` and I deployed it to heroku. When I make change to an existing tiddler or add a new one, it's not saved to the file system. The only file that's changed is `$:/StoryList`. Nothing gets logged by the server after the initial logs about loading plugins.

**Comment by bmann** [comment]
@adamschmideg yes this is not complete at the moment. Heroku doesn’t have a file system. This is me experimenting with what TiddlyWiki + Heroku looks like.

**Comment by jeprecated** [comment]
Any plans to make this more complete? I too am looking for a solution to this

**Comment by bmann** [comment]
@ohare93 I'm not working on this approach directly right now.

The Git Saver method works well with Github / Github Pages, and is the easy way to get started now, I wrote a blog post that has links to what I did https://blog.fission.codes/tiddlywiki-as-the-original-serverless/

And here's a work-in-progress template for the Drift Tiddlywiki that has instructions on Git Save mode https://github.com/bmann/drift-tiddlywiki-template

What I would like to see is individual tiddler mode, with multi user, and encryption :)

@ohare93 if you can post some more details on what you're trying to accomplish, that would be helpful!

**Comment by jeprecated** [comment]
> I'm not working on this approach directly right now.

@bmann Ah well, no problem. Thanks for the detailed reply and the link.

I believe I will be going down the host a TiddlyWiki on a Raspberry Pi route then :+1: all I am looking for is a self owned TiddlyWiki setup that I can access from anywhere, but that is also secure with the possibility of allowing other users in. Hosting it as a permanent server in node.js may be the answer I've been looking for.

https://www.woolie.co.uk/article/tiddlywiki-self-hosted/
https://fedwiki.frankmcpherson.net/view/tiddlyserver-on-raspberry-pi

**Comment by bmann** [comment]
Ah, the multi-users stuff can be tricky. But, if you’re on RPi, you can just use the NodeJS server and it just works.

That’s how this repo is setup, and it will work locally because you’ve got a file system.
